docs/basics/101-103-modify.rst

```diff
@@ -137,3 +137,11 @@ We can see that the history can not only show us the commit message attached to
 a commit, but also the precise change that occurred in the text file in the commit.
 Additions are marked with a ``+``, and deletions would be shown with a leading ``-``.
 That's quite neat, isn't it?
+
+.. findoutmore:: ``git log`` has many more useful options
```

Comment:
```suggestion
.. findoutmore:: git log has many more useful options
```

The header of these directive does not support RST markup as far as I'm aware

Reply:
Ah, will remove then